packages/models-library/tests/test_volumes.py

```diff
@@ -0,0 +1,16 @@
+# pylint: disable=redefined-outer-name
+
+import pytest
+from models_library.volumes import VolumeState, VolumeStatus
+from pytest import FixtureRequest
+
+
+@pytest.fixture(params=VolumeStatus)
+def status(request: FixtureRequest) -> VolumeStatus:
+    return request.param
+
+
+def test_volume_state_equality(status: VolumeStatus):
+    assert VolumeState(status=status) == VolumeState(status=status)
```

Comment:
what is the use of such a test?

Reply:
Each time `VolumeState` is created the `last_changed` field is different since it calls the constructor.
This checks that the field is not included in the equality. Let me add some comments here.